index.js

```diff
@@ -4,16 +4,16 @@ const discovery = require('hyperdiscovery')
 
 module.exports = function (archiveOrKey, opts) {
   var archive = archiveOrKey.key ? archiveOrKey : createArchive(archiveOrKey)
+  var feed = (archive.content) ? archive.content : archive
   var swarm = discovery(archive, opts)
+
   archive.open(function () {
```

Comment:
not sure, my knowledge of `hyperdrive` is limited - the logic is the same as before tho so I'd rather not deal with changing it in this patch hmm